src/components/layout/header.tsx

```
@@ -24,65 +184,246 @@ export const Header = () => {
+              </NavLink>
+              <NavLink
+                to={ROUTES.ADMIN_DASHBOARD}
+                className="whitespace-nowrap text-sm font-medium text-gray-600 transition hover:text-gray-900"
+              >
+                Admin
+              </NavLink>
+            </nav>
+
+            <div className="flex items-center gap-3">
+              <NotificationMenu />
+              <button className="rounded-lg p-1.5 transition hover:bg-gray-100">
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)